app/popup.html

```diff
@@ -2,93 +2,154 @@
 <html lang="en" ng-app="themerApp" ng-csp>
 
 <head>
-    <meta charset="utf-8">
-    <!-- build:css styles/popup-vendor.css -->
-    <!-- bower:css -->
-    <!-- endbower -->
-    <!-- endbuild -->
-    <!-- build:css styles/main.css -->
-    <link href="styles/main.css" rel="stylesheet">
-    <link href="bower_components/angular/angular-csp.css" rel="stylesheet">
-
-    <!-- Latest compiled and minified CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-    <!-- endbuild -->
-
-    <!-- build:js scripts/popup-vendor.js -->
-    <!-- bower:js -->
-    <!-- endbower -->
-    <!-- endbuild -->
-
-    <!-- build:js scripts/popup.js -->
-    <script src="scripts/popup.js"></script>
-    <script src="scripts/background.js"></script>
-    <!-- endbuild -->
-    <script src="bower_components/angular/angular.js"></script>
-    <script src="scripts/app.js"></script>
+	<meta charset="utf-8">
+
+	<!-- build:css styles/popup-vendor.css -->
+	<!-- bower:css -->
+	<!-- endbower -->
+	<!-- endbuild -->
+	<!-- build:css styles/main.css -->
+	<link href=https://cdn.jsdelivr.net/picnicss/6.3.2/picnic.min.css>
+	<link href="styles/main.css" rel="stylesheet">
+	<link href="bower_components/angular/angular-csp.css" rel="stylesheet">
+	<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> -->
+	 <!-- endbuild -->
+
+	<!-- build:js scripts/popup-vendor.js -->
+	<!-- bower:js -->
+	<!-- endbower -->
+	<!-- endbuild -->
+
+	<!-- build:js scripts/popup.js -->
+	<script src="scripts/popup.js"></script>
+	<script src="scripts/background.js"></script>
+	<!-- endbuild -->
+	<script src="bower_components/angular/angular.js"></script>
+	<script src="scripts/app.js"></script>
+	<link rel="stylesheet" href="picnic.css">
```

Comment:
What is the difference between cdn stylesheet of picnic.min.css on line 12 and picnic.css on line 29? If you're already using a cdn, why include a seperate remote stylesheet that has the same content?